clab/deploy_options.go

```diff
@@ -0,0 +1,79 @@
+package clab
+
+import (
+	"math"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/tklauser/numcpus"
+)
+
+type DeployOptions struct {
+	reconfigure    bool
+	skipPostDeploy bool
+	graph          bool
+	maxWorkers     uint
+	exportTemplate string
+}
+
+func NewDeployOptions(maxWorkers uint) (*DeployOptions, error) {
+	d := &DeployOptions{
+		maxWorkers: math.MaxUint,
+	}
```

Comment:
seems like this is not needed
setting maxworkers to the max value is not going to be useful in any case since it is reset in the initWorkerCount

Reply:
Done